lbrynet/core/utils.py

```diff
@@ -9,6 +9,8 @@
 
 import pkg_resources
 
+from twisted.internet import reactor
```

Comment:
if twisted.internet.reactor was imported before:

https://github.com/lbryio/lbry/blob/267c6cbaca739fdce06c51b3251eecb75b9f31f5/packaging/osx/lbry-osx-app/lbrygui/main.py#L3-L3

I suppose it can't be a problem if we aren't using pyobjc for the app